lambda/js/src/main/scala/feral/lambda/ContextPlatform.scala

```diff
@@ -27,14 +27,14 @@ private[lambda] trait ContextCompanionPlatform {
       context.functionName,
       context.functionVersion,
       context.invokedFunctionArn,
-      context.memoryLimitInMB.toInt,
```

Comment:
Netlify seems to pass an int here, but the facade uses a string.

Reply:
Follow-up: inconsistency in the simulator/emulator, we'll support both.